app/Repositories/FamilyRepository.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace App\Repositories;
+
+use App\Exceptions\AuthorizedUserNotFoundException;
+use App\Exceptions\CarNotAttachedToFamilyException;
+use App\Exceptions\FamilyDetailsNotFoundException;
+use App\Exceptions\FamilyNotDeletedException;
+use App\Exceptions\FamilyNotSavedToDatabaseException;
+use App\Exceptions\FamilyNotUpdatedException;
+use App\Models\Family;
+use App\Models\UserModel;
+use Illuminate\Database\Eloquent\Collection;
+
+class FamilyRepository
+{
+    protected ?Family $family;
```

Comment:
nie lepiej $family = null ?

Reply:
Rozumiem, że chodzi o wartość domyślną - można dopisać, aczkolwiek przy obecnej logice działania repozytorium wydaje mi się, że nic to nie zmieni.